lib/features/family/shared/widgets/layout/givt_bottom_sheet.dart

```diff
@@ -0,0 +1,123 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+import 'package:givt_app/features/family/features/impact_groups/cubit/impact_groups_cubit.dart';
+import 'package:givt_app/shared/widgets/buttons/givt_elevated_button.dart';
+import 'package:givt_app/shared/widgets/buttons/givt_elevated_secondary_button.dart';
+
+class GivtBottomSheet extends StatelessWidget {
+  const GivtBottomSheet({
+    required this.title,
+    required this.icon,
+    required this.content,
+    this.headlineContent,
+    this.primaryButton,
+    this.secondaryButton,
+    this.closeAction,
+    super.key,
+  });
+
+  final String title;
+  final Widget icon;
+  final Widget content;
+  final Widget? headlineContent;
+
+  final GivtElevatedButton? primaryButton;
+  final GivtElevatedSecondaryButton? secondaryButton;
+
+  final VoidCallback? closeAction;
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<ImpactGroupsCubit, ImpactGroupsState>(
```

Comment:
There's no need for a blocbuilder here as the state is not being used. Where you planning on using the state?

Reply:
nope :) I'll remove that part indeed